server/src/test/java/org/elasticsearch/index/mapper/FieldAliasMapperValidationTests.java

```diff
@@ -158,7 +158,7 @@ public void testFieldAliasWithDifferentNestedScopes() {
     }
 
     private static FieldMapper createFieldMapper(String parent, String name) {
-        return new BooleanFieldMapper.Builder(name, ScriptCompiler.NONE, Version.CURRENT).build(new MapperBuilderContext(parent, false));
+        return new BooleanFieldMapper.Builder(name, ScriptCompiler.NONE,false, Version.CURRENT).build(new MapperBuilderContext(parent, false));
```

Comment:
nit: another spotless formatting thingy here

Reply:
The above formatting issue has been fixed as well